packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -379,6 +384,15 @@ export class ExternalRequest {
       ) {
         continue
       }
+
+      if (
```

Comment:
This is ensuring we don't map wrong fields when both primary and foreign tables have the same field names